app/src/main/java/com/onelist/dao/IItemDAO.kt

```diff
@@ -6,6 +6,7 @@ import retrofit2.Call
 import retrofit2.http.GET
 
 interface IItemDAO {
-    @GET("") //TODO: add rest of retrofit link for items
```

Comment:
Wired Retrofit to endpoint to consume Item JSON. Change this URL to master if merged to main